packages/lesswrong/lib/rateLimits/constants.ts

```diff
@@ -48,114 +49,93 @@ const LW: {POSTS: PostAutoRateLimit[], COMMENTS: CommentAutoRateLimit[]} = {
   // 2 posts per week rate limits
     {
       ...timeframe('2 Posts per 1 weeks'),
-      karmaThreshold: 4,
       rateLimitType: "newUserDefault",
+      isActive: user => user.karma < 5,
       rateLimitMessage: `Users with less than 5 karma can write up to 2 posts a week.<br/>${lwDefaultMessage}`,
     }, 
   // 1 post per week rate limits
     {
       ...timeframe('1 Posts per 1 weeks'),
-      karmaThreshold: -1,
+      isActive: user => user.karma <= -3,
       rateLimitMessage: `Users with -3 or less karma can post once per week.<br/>${lwDefaultMessage}`
     }, 
     {
       ...timeframe('1 Posts per 1 weeks'),
-      last20PostKarmaThreshold: -15,
-      downvoterCountThreshold: 4,
+      isActive: (user, features) => features?.last20PostKarma <= -15 && features?.postDownvoterCount >= 4,
```

Comment:
I predict that rate limit `isActive` logic with multiple conditions will be hard to read on a single line and advocate for pre-emptively splitting it across multiple lines:
```
isActive: (user, features) => (
  features?.last20PostKarma <= -15 &&
  features?.postDownvoterCount >= 4
),
```